client.go

```diff
@@ -417,3 +425,28 @@ type DeviceLinkGroupLink struct {
 	SourceZoneCode       *string
 	DestinationZoneCode  *string
 }
+
+//ClusterDetails describes Network Edge cluster device details
+type ClusterDetails struct {
+	ClusterName        *string
+	NumOfNodes         *int
```

Comment:
Could you help me understand the differences between these implementation changes and what is described in the spec? Ideally the spec should reflect the same fields, unless I am looking at the wrong endpoint and schema: https://developer.equinix.com/catalog/network-edgev1#tag/Virtual-Device

Reply:
🤦 I see the relevant documentation in the link you provided in the PR description https://developer.equinix.com/docs/launch-cluster-device.

The generated catalog spec is out-of-date, but I'll verify your changes against the API Guide.